internal/brokers/broker_test.go

```diff
@@ -14,6 +15,24 @@ import (
 	"github.com/ubuntu/authd/internal/testutils"
 )
 
+var supportedLayouts = map[string]map[string]string{
+	"required-value": {
+		"type":  "required-value",
+		"value": "required:value_type, other_value_type",
```

Comment:
you have spaces here, we don’t really return this default, so maybe trim that? If we want to assert one the support, this is an additional test case (and so layout) maybe?

Reply:
Yes, I didn't think about it but trimming after the split is easy enough to do and be more permissive. Then, having a test for it (I think it’s small enough to get into that PR)

Reply:
Do we want a test function just for it? Or should we attach it to the GAM tests (it could be a simple test case)?

Reply:
oh testcase, definitively.